src/doc/template/tmpl/layout.tmpl

```diff
@@ -161,13 +161,13 @@
             </span>
         </footer>
 
-        <script src="scripts/docstrap.lib.js"></script>
-        <script src="scripts/toc.js"></script>
+        <script src="/britecharts/scripts/docstrap.lib.js"></script>
```

Comment:
I don't think that this will work once we push it into our github pages, as the parth would be `/docs/scripts`